contrib/core/actions/pause.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  name: "pause"
+  runner_type: "run-python"
+  description: "Action to Pause workflow"
```

Comment:
well it will pause "everything", not just a workflow :) Pls craft a description that also hints an intended use.